tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -49,6 +50,18 @@ limitations under the License.
 
 namespace tensorflow {
 
+/// This table contains for each node name descriptors on which
+/// hardware  to check whether we should rewrite the operations
```

Comment:
remove extra space between "hardware" and "to"

Reply:
Thanks for spotting it. Fixed in [59cc901](https://github.com/tensorflow/tensorflow/commit/59cc9014114ac5416e8980759ac6d105b9ae116d)